scripts/e2e-minikube.sh

```diff
@@ -3,6 +3,10 @@
 readonly usage="Usage: e2e-minikube.sh --test-tag <test-id>"
 readonly SERVICE_ACCOUNT="travis-tests"
 
+readonly RUNASUSER="\n  securityContext:\n    runAsUser: 1001"
+readonly APPIMAGE='applicationImage:\s'
+readonly IMAGES=('k8s.gcr.io\/pause:2.0' 'navidsh\/demo-day' 'icr.io\/appcafe\/open-liberty\/samples\/getting-started')
```

Comment:
getting-started should run as non-root user  I believe, so wondering the reason for including that here.